test(#3807): Add disabled test for XMIR bytes with anonymous abstract object

volodya-lombrozo opened the pull request:
This pull request introduces a new test case in `EoSyntaxTest` to verify the correct compilation of bytes to XMIR with an anonymous abstract object. The test is currently disabled due to an existing issue with the XMIR format, which is missing an anonymous abstract object for bytes. 

Related to #3807

volodya-lombrozo commented:
@maxonfjvipon Is this what we want to achieve in the end?

maxonfjvipon commented:
@volodya-lombrozo but maybe yes, you're right about
```
bytes > app
  2A-
```

It should be:
```xml
<o base="Q.org.eolang.bytes" name="app">
  <o base="Q.org.eolang.bytes">
    <o>2A-</o>
  </o>
</o>
```

maxonfjvipon commented on eo-parser/src/test/java/org/eolang/parser/EoSyntaxTest.java:
```diff
@@ -384,4 +384,45 @@ void printsSyntaxWithComments() throws IOException {
+            String.join(
+                "\n",
+                "XMIR should contain the anonymous abstract object for bytes. Expected: ",
+                "<o base='bytes' name='app'>",
```
@volodya-lombrozo let's use FQNs here like `Q.org.eolang.bytes`

maxonfjvipon commented on eo-parser/src/test/java/org/eolang/parser/EoSyntaxTest.java:
```diff
@@ -384,4 +384,45 @@ void printsSyntaxWithComments() throws IOException {
+     */
+    @Test
+    @Disabled
+    void compilesXmirBytesWithAnonymousAbstractObject() throws IOException {
```
@volodya-lombrozo let's add one more test with `string` or `number` where EO code is like:
```
# No comments.
0 > zero
```

volodya-lombrozo commented:
@maxonfjvipon I've moved the test from Java to eo packs. Now they look a bit prettier. Also I've added tests for `number` and `string`. What do you think?

maxonfjvipon reviewed (approved):
@volodya-lombrozo LGTM! Thanks

volodya-lombrozo commented:
@yegor256 Could you merge that changes, please?

yegor256 commented:
@volodya-lombrozo once we "resolve" all conversations above, GitHub will merge this one automatically

volodya-lombrozo commented:
@yegor256 It's a bit risky to merge after all the conversations are resolved because I can resolve them myself.
